.github/workflows/e2e.yml

```diff
@@ -39,6 +41,8 @@ jobs:
         env:
           PORT: 4000
           DATABASE_URL: postgres://postgres:postgres@postgres:5432/postgres
+          CI: false
+          NODE_ENV: development
```

Comment:
Not disagreeing with this choice, but curious why we are choosing to use dev rather than prod